docs/experimental-plugin-preview.md

```
@@ -0,0 +1,160 @@
+# Option A: Use the helper script
+./bin/preview-plugin.sh my-new-plugin --publish
+
+# Option B: Remove "experimental": true from plugin.json
+# The next sync/deploy will register it as visibleToUsers: true
+```
+
+## Helper Script Reference
+
+`bin/preview-plugin.sh` automates the full workflow:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)